djautotask/api.py

```diff
@@ -353,8 +355,7 @@ def get_headers(self, method):
             headers['Secret'] = self.password
         if self.integration_code:
             headers['ApiIntegrationCode'] = self.integration_code
-        if self.impersonation and self.impersonation_id \
-                and method.lower() != 'get':
+        if self.impersonation_id and method.lower() != 'get':
```

Comment:
Seems like an extra step to convert to lowercase to compare with `get`, pretty standard to refer to request methods in captitals, like GET, POST, PATCH, etc.